tests/rand.rs

```diff
@@ -0,0 +1,22 @@
+#![cfg(feature = "rand")]
+
+use evalexpr::*;
+
+fn assert_expr(expr: &str) {
+    assert_eq!(eval(expr), Ok(Value::Boolean(true)))
+}
+
+#[test]
+fn test_random() {
+    for _ in 0..100 {
+        assert_expr("random() != random()");
```

Comment:
Don't you think not including this test introduces a much higher risk than two random 64-bit numbers being equal ?

Reply:
If you have ten million users, and each one runs the tests 100 times every day, the probability it will fail for someone in the next 100 year is [around one in a million](https://www.wolframalpha.com/input?i=%281-1%2F%282**64%29%29**%281e7*100*365*100%29)

Reply:
Well, the proper way to do this is to allow the context to carry the random generator, and then for the tests use one that does not repeat numbers if called only a limited amount of times. 

You're right that it is extremely unlikely that this test actually ever fails, but it doesn't feel very good to have tests that may fail in rare circumstances.

If you could add a random generator to the context trait that would be great, however that is probably a bit more work. If you want to keep the test in there like it is, please add a comment that this may fail and the user should try again if it does. If using a normal `assert!` instead of `assert_expr` there could also be a customised error message for this test.